src/Tribe/blue.code-workspace

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file should be ignored.

Reply:
removed.

Reply:
Also added that file type to .gitignore to prevent that in the future (VSCode config)